### PR TITLE
Create 10.2024.4 - Appeals Committee 

### PR DESCRIPTION
**Pull request by Epride**

### CODE REVIEW

File: documents/motions/10.2024.4 - Appeals Committee.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+   7. Additional duties as directed by the WCA Board.
+2. All WCA Staff and Registered Speedcubers must fully cooperate with investigations by the WCA Appeals Committee.
+3. In case the WCA Appeals Committee is not constituted or for any reason non-functional, the WCA Board may appoint select WCA Staff to handle cases of appeal.
+4. The WCA Appeals Committee will take the role of WCA Integrity Committee in case the WCA Integrity Committee is not constituted or for any reason non-functional.
+      1. For any period where the WCA Appeals Committee assumes the role of the WCA Integrity Committee the WCA Board may appoint select WCA Staff to handle cases of appeal.
+5. Reducing conflicts of roles:
+   1. If all bodies are constituted and functional there must not exist an overlap in members of the WCA Board, WCA Integrity Committee, and WCA Appeals Committee unless the Board is temporarily required to lead, due to that Committee not having a Leader.
+6. Eligibility for membership:
+   1. To be eligble to serve as a member of the WCA Appeals Committee the candidate must have served a period of at least 12 consecutive months as a member of WCA Staff.
```
Comment (KerrieJarman) [inline documents/motions/10.2024.4 - Appeals Committee.md:34]: Does it matter when that experience is from?  Should it be recent experience or does it not matter?  Personally I don't think it matters all that much but it is worth considering

File: documents/motions/10.2024.4 - Appeals Committee.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+   2. Performing independent investigations regarding appealed incidents.
+   3. Making decisions on disciplinary actions in relation to appealed incidents.
+   4. Informing the WCA Board on appeals, investigations, decisions, and disciplinary actions.
+   5. Publishing public information on the appeals, investigations, decisions, and disciplinary actions where necessary.
+   6. Managing an administration of the appeals, investigations, the reports, the decisions, and the disciplinary actions.
+   7. Additional duties as directed by the WCA Board.
+2. All WCA Staff and Registered Speedcubers must fully cooperate with investigations by the WCA Appeals Committee.
+3. In case the WCA Appeals Committee is not constituted or for any reason non-functional, the WCA Board may appoint select WCA Staff to handle cases of appeal.
+4. The WCA Appeals Committee will take the role of WCA Integrity Committee in case the WCA Integrity Committee is not constituted or for any reason non-functional.
+      1. For any period where the WCA Appeals Committee assumes the role of the WCA Integrity Committee the WCA Board may appoint select WCA Staff to handle cases of appeal.
```
Comment (KerrieJarman) [inline documents/motions/10.2024.4 - Appeals Committee.md]: "select" is a bit clunky, maybe specific is a better word

File: documents/motions/10.2024.4 - Appeals Committee.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      1. The Board of Appeal shall be made up of three members of the WCA Appeals Committee.
+      2. The Leader of the WCA Appeals Committee determines the members of the Board of Appeal for each dispute.
+      3. Members of the WCA Appeals Committee who have a conflict of interest for a specific dispute shall be ineligible from selection to that specific Board of Appeal.
+      4. The Board of appeal shall have all the rights and duties assigned to the WCA Appeals Committee and shall have the full right to represent the committee in the appeals process.
+   2. Performing independent investigations regarding appealed incidents.
+   3. Making decisions on disciplinary actions in relation to appealed incidents.
+   4. Informing the WCA Board on appeals, investigations, decisions, and disciplinary actions.
+   5. Publishing public information on the appeals, investigations, decisions, and disciplinary actions where necessary.
+   6. Managing an administration of the appeals, investigations, the reports, the decisions, and the disciplinary actions.
```
Comment (KerrieJarman) [inline documents/motions/10.2024.4 - Appeals Committee.md]: "Managing an administration" seems odd, Maybe just "Administering" or "Maintaining records".  but tbh not sure of the intent of this sentence.  Is it necessary to codify that they will keep records of their work?

File: documents/motions/10.2024.4 - Appeals Committee.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+**Intent:** Rights and duties of the WCA Appeals Committee  
+**Submitted by:** WCA Board  
+**Date:** TBC, 2024  
+
+# Motion
+
+The WCA Appeals Committee is an Advisory Committee of the WCA.
+
+1. The WCA Appeals Committee has the following rights and duties:
+   1. Forming the Board of Appeal to carry out the appeals processes described in [WCA Motion : Disputes](wca{documents/motions/16.2021.1 - Disputes.pdf}).
```
Comment (KerrieJarman) [inline documents/motions/10.2024.4 - Appeals Committee.md]: I think there needs to be something here that acknowledges that the Board of Appeal is a temporary structure that exists just for the life of this specific appeal.  Also that there may by multiple of them.  It is not just "the" Board of Appeal.  There could be multiple running concurrently.

File: documents/motions/10.2024.4 - Appeals Committee.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      1. The Board of Appeal shall be made up of three members of the WCA Appeals Committee.
+      2. The Leader of the WCA Appeals Committee determines the members of the Board of Appeal for each dispute.
+      3. Members of the WCA Appeals Committee who have a conflict of interest for a specific dispute shall be ineligible from selection to that specific Board of Appeal.
+      4. The Board of appeal shall have all the rights and duties assigned to the WCA Appeals Committee and shall have the full right to represent the committee in the appeals process.
```
Comment (KerrieJarman) [inline documents/motions/10.2024.4 - Appeals Committee.md]: Consistency of capitalisation needed

File: documents/motions/10.2024.4 - Appeals Committee.md (hunk 1 above)
Comment (Epride) [reply documents/motions/10.2024.4 - Appeals Committee.md:34]: I have left it without restriction for now just to indicate that 12 months is required but not where specifically. I think this gives the leader some flexibility while noting this position is not for members who have little to no idea on the WCA's internal structure.

File: documents/motions/10.2024.4 - Appeals Committee.md (hunk 4 above)
Comment (KerrieJarman) [reply documents/motions/10.2024.4 - Appeals Committee.md]: Suggested wording "Forming a Board of Appeal for a dispute.  The Board of Appeal is a temporary team to carry out the appeals processes described in ..."

File: documents/motions/10.2024.4 - Appeals Committee.md (hunk 3 above)
Comment (EdHollingdale) [reply documents/motions/10.2024.4 - Appeals Committee.md]: My view is it is 1) Administering of the outcomes of appeals (e.g. tracking that a ban is put in force), 2) maintaining a documents record of appeals etc. I have proposed updated wording.

File: documents/motions/10.2024.4 - Appeals Committee.md (hunk 4 above)
Comment (EdHollingdale) [reply documents/motions/10.2024.4 - Appeals Committee.md]: I have incorporated changes for this.

File: documents/motions/10.2024.4 - Appeals Committee.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+   6. Administering and maintaining records of the appeals, investigations, the reports, the decisions, and the disciplinary actions.
+   7. Additional duties as directed by the WCA Board.
+2. All WCA Staff and Registered Speedcubers must fully cooperate with investigations by the WCA Appeals Committee.
+3. In case the WCA Appeals Committee is not constituted or for any reason non-functional, the WCA Board may appoint select WCA Staff to handle cases of appeal.
+4. The WCA Appeals Committee will take the role of WCA Integrity Committee in case the WCA Integrity Committee is not constituted or for any reason non-functional.
+      1. For any period where the WCA Appeals Committee assumes the role of the WCA Integrity Committee the WCA Board may appoint specific WCA Staff to handle cases of appeal.
+5. Reducing conflicts of roles:
+   1. If all bodies are constituted and functional there must not exist an overlap in members of the WCA Board, WCA Integrity Committee, and WCA Appeals Committee unless the Board is temporarily required to lead, due to that Committee not having a Leader.
```
Comment (CarterKoala) [inline documents/motions/10.2024.4 - Appeals Committee.md:31]: Have there been any considerations for conflict of roles between WRC and Appeals committee? I'm not sure if just counting it as a conflict of interest as in 1.1.4 is enough - it probably doesn't look good to an outsider to see a member on both.

Comment (EdHollingdale) [reply documents/motions/10.2024.4 - Appeals Committee.md:31]: Great point!

**Pull request merged**